src/js/controllers/tab-send.controller.js

```diff
@@ -61,6 +61,49 @@ angular.module('copayApp.controllers').controller('tabSendController', function(
     });
   });
 
+  $scope.closePopover = function() {
+    $scope.popover.hide();
+  };
+
+  $scope.$on('$destroy', function() {
+    $scope.popover.remove();
+  });
+
+  $scope.confirmOpenapAddress = function(address, coin){
+    $scope.popover.remove();
+    var params = sendFlowService.state.getClone();
+    params.data = address;
+    params.coin = coin;
+    sendFlowService.start(params, function onError() {
+      return
+    });
+  }
+  
+  $scope.resolveOpencapAlias = function(alias){
+    opencapService.getAddress(alias)
+    .then(result => {
+      if (typeof $scope.fromWallet !== 'undefined'){
+        if ($scope.fromWallet.coin === 'bch'){
+          result.addresses = {bch: result.addresses.bch};
+        }
+        if ($scope.fromWallet.coin === 'btc'){
+          result.addresses = {btc: result.addresses.btc};
+        }
+      }
+
+      $scope.opencapAddresses = result.addresses;
+      $scope.opencapDnssec = result.dnssec;
+      $ionicPopover.fromTemplateUrl('templates/popoverOpencapSend.html', {scope: $scope})
+      .then(function(popover) {
+          $scope.popover = popover;
+          popover.show(angular.element(document.querySelector('#search-input')))
+      });
+    })
+    .catch(status => {
```

Comment:
Now :  `=>`  : lambda expression is not supported for old javascript engine.
Should be : 
```
function onCatch() {
}
```